recipes/default.rb

```diff
@@ -17,20 +17,24 @@
 # limitations under the License.
 
 node.default['postfix']['mail_type'] = 'client'
-node.default['postfix']['main']['myorigin'] = '$mydomain'
-node.default['postfix']['main']['smtpd_use_tls'] = 'no'
+node.default['osl-postfix']['main']['myorigin'] = '$mydomain'
+node.default['osl-postfix']['main']['smtpd_use_tls'] = 'no'
```

Comment:
Given we're now using our own attributes, it might make sense to move all of these to the attributes file instead.